resources/tech/otel/observability-2-0-honeycomb.md

```diff
@@ -0,0 +1,23 @@
+# Observability 2.0 (Honeycomb Blog)
+
+Resource type: Blog Post
+
+https://www.honeycomb.io/blog/one-key-difference-observability1dot0-2dot0
+
+## What it’s about
+
+Honeycomb lays out the core distinction between “Observability 1.0” (centered on dashboards, metrics, and alerting) and “Observability 2.0” (centered on wide, structured events that allow flexible, ad-hoc exploration). The post argues that OTel is enabling this shift across the industry.
+
+## Why it’s worth reading
+
+This is the canonical articulation of what “Observability 2.0” means. Many teams still think observability is just monitoring with more dashboards; this article helps reframe it as a fundamentally different practice.
+
+## Pause and Ponder
+
+- Are we treating observability as pre-built dashboards, or as a flexible tool for answering new questions?
+- How could wide structured events help us see connections we’re missing today?
+- Where in our org would shifting from “1.0” to “2.0” make the biggest cultural impact?
+
+## Takeaway
+
+Observability 2.0 isn’t just more data — it’s a mindset shift. Wide structured events let engineers ask questions they didn’t anticipate, turning telemetry from a static dashboard into a dynamic conversation.
```

Comment:
Here I'd say there are 2 practical takaways to o11y 2.0:
1. Broaden the field of vision - add more (even sometimes seemingly redundant) attributes - don't be stingy there. If you need to save money, sample wide telementry, don't make it sparse or skinny. or use open-source Visualizers.
2. Bring the whole picture together as a single feed.

(Practically speaking, it doesn't mean "only use traces") - ... maybe prefer traces though for a lot of things.

This allows you to zoom in when needed, see broadly when needed, easily make connections that otherwise you might have missed.